lib/jellyfish_web/views/room_view.ex

```diff
@@ -0,0 +1,23 @@
+defmodule JellyfishWeb.RoomView do
+  use JellyfishWeb, :view
+  alias JellyfishWeb.RoomView
+  alias JellyfishWeb.PeerView
+  alias JellyfishWeb.ComponentView
+
+  def render("index.json", %{rooms: rooms}) do
+    %{data: render_many(rooms, RoomView, "room.json")}
+  end
+
+  def render("show.json", %{room: room}) do
+    %{data: render_one(room, RoomView, "room.json")}
+  end
+
+  def render("room.json", %{room: room}) do
+    %{
+      id: room.id,
+      config: room.config,
+      components: ComponentView.render_dict(room.components),
```

Comment:
Couldn't we use `ComponentView.render("index.json", %{peers: peers}`. Creating separate function to do the same thing seems unnecessary (I'm not sure if it does the same thing tho, maybe I don't understand something).

Reply:
I think it could be a little bit problematic that the components field is a dictionary, but I can try to change it.